homeassistant/components/nut/sensor.py

```diff
@@ -35,9 +35,12 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
     unique_id = pynut_data[PYNUT_UNIQUE_ID]
     status = coordinator.data
 
-    enabled_resources = [
-        resource.lower() for resource in config_entry.data[CONF_RESOURCES]
-    ]
+    enabled_resources = []
+    if CONF_RESOURCES in pynut_data:
```

Comment:
I dont think we need to keep this. The entity registry already remembers the enabled status even if we change the default.

Reply:
I keep this to avoid the situation for user that migrate from HA version pre 2021.11.x should have more sensors enabled, because with previous versions only resources selected during initial configuration or in option flow was created. If we don't want to care about this I can remove.

Reply:
With the device categories I think it's ok to have the sensors be enabled for those users. They can disable the sensors if they want to.